### DISCUSSION TITLE
Is it possible to use old ckeditor v4 with latest backpack version?

### DISCUSSION THREAD

**Discussion by jimiero** [Laravel-Backpack/community-forum#1261] [Q&A (Help)]
Hello,

I've recently updated a project to latest version of backpack, but now seems it uses ckeditor version 5 and editor plugins it was using before are not working anymore

Is it possible to make backpack to still use old v4 of ckeditor ?

Thanks

[a comment by an automated account is not shown]

**Comment by jimiero** [comment]
there is no such ckeditor.blade.php file inside backpack crud version 5 branch any help would be appriciated

**Comment by jimiero** [comment]
@tabacitu any help here please?

**Comment by jimiero** [comment]
@pxpm do you have some more details on how to achieve this?

**Comment by karandatwani92** [comment]
Hey @jimiero 

Sorry for late reply. Yes, you can use `Ckeditor 4` or any other `editor` of your choice but you have to [create a Backpack custom field](https://backpackforlaravel.com/docs/6.x/crud-fields#creating-a-custom-field-type-1) using that JS lib.
